package.json

```diff
@@ -22,12 +22,12 @@
     ]
   },
   "scripts": {
-    "integration": "tsc && cd ../azure-rest-api-specs/ && node ../avocado/bin/cli.js",
-    "cli": "tsc && node ./bin/cli.js",
-    "tsc": "tsc",
+    "integration": "npx tsc && cd ../azure-rest-api-specs/ && node ../avocado/bin/cli.js",
```

Comment:
NPX will use a global temporary install on each run. Are you sure that's what you're trying to do? This will also be unpinned from the package.json version

Reply:
Thanks. I misunderstand it. Because if i run `tsc` locally, tsc use global version not local package version and it reports compile error (avocado can't be compiled in typescript which version higher than 3.6.0).  It seems in npm scripts, run tsc it will use local package version.

Reply:
No worries, but it looks like this line still has the `npx` 😉

Reply:
updated.